### PR TITLE
Drop MacroTools dependency

### PR DESCRIPTION
**Pull request by oxinabox**
Its not worth depending on MacroTools and thus anything it depends on (DataStructures & OrderedCollections)

For this simple function.

I also think this implementation of the function is better,

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -1,11 +1,8 @@
 name = "MuladdMacro"
 uuid = "46d2c3a1-f734-5fdb-9937-b9b9aeba4221"
-
-[deps]
-MacroTools = "1914dd2f-81c6-5fcd-8719-6d5c9610ff09"
+version = "0.2.2"
```
Comment (oxinabox) [inline Project.toml]: Last release was 0.2.1
so this is good to tag once this is merged

**Comment by devmotion** [comment]
> I also think this implementation of the function is better,

Do you mean this implementation of `postwalk` should be used by MacroTools as well?

**Comment by oxinabox** [comment]
>  Do you mean this implementation of postwalk should be used by MacroTools as well?


I was mostly being facecious, because its a 2 line function so how can one do it "better".
Though I do fine MacroTool's harder to read because it abstracts `prewalk` and `postwalk` into a single `walk` inner function.

**Comment by devmotion** [comment]
Yes, I agree that your implementation is definitely easier to understand :+1:

**Comment by devmotion** [comment]
@ChrisRackauckas, can you merge the PR? I don't have write access.

**Comment by ChrisRackauckas** [comment]
oh you don't? I thought you did.

**Comment by devmotion** [comment]
I was also surprised :stuck_out_tongue:

**Pull request merged**